docs/reference/changelog.rst

```diff
@@ -16,6 +16,17 @@ Changelog
 
   For a complete list of commits, check out the `X.Y.Z`_ release on GitHub.
 
+.. _release-2.24.0:
+
+2.24.0 (unreleased)
+-------------------
+
+New features:
+
+- Add a new version of the ``PythonPlugin``. This new version is not registered by
+  default as it is still experimental and has a different set of requirements and
+  behaviors when compared to the default plugin for Python projects.
```

Comment:
```suggestion
- Add an experimental new v2 ``PythonPlugin``. This new version is not registered by
  default has a different set of requirements and  when compared to the default plugin
  for Python projects.
```

If it weren't going to be shot down I'd ask for 72 pt font saying "THIS NEW PLUGIN IS EXPERIMENTAL DO NOT EXPECT BACKWARDS COMPATIBILITY AND DO EXPECT IT TO EAT YOUR CAT"

Reply:
That's a good idea. I did a slight rewording of this^:
<img width="841" height="265" alt="image" src="https://github.com/user-attachments/assets/776e32ee-ad4e-4fc1-8acd-919febb5ffa9" />